R/ospsuite-env.R

```diff
@@ -60,6 +54,11 @@ ospsuiteEnv$isPKSimLoaded <- FALSE
 # NetTask `DimensionTask` cached for performance benefits. Created the first time it is requested.
 ospsuiteEnv$dimensionTask <- NULL
 
+#' Names of the settings stored in ospsuiteEnv. Can be used with `getOSPSuiteSetting()`
+#' @include utilities.R
+#' @export
+ospsuiteSettingNames <- enum(names(ospsuiteEnv))
```

Comment:
Worth logging this new export in `NEWS.md`.

Reply:
Would add it when we have the separation between user-defined "global" settings and "read-only".